flake.nix

```diff
@@ -153,7 +153,8 @@
 
         workspaceDeps = craneLib.buildDepsOnly (commonArgs // {
           src = filterWorkspaceDepsBuildFiles ./.;
-          pname = "fedimint-dependencies";
+          pname = "workspace-deps";
```

Comment:
Added `pname` so it's easier to track each step when looking at the outputs. Otherwise they all have something generic like `package-0.0.1` or something like that.